Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocColumnRendererBase.cs

```diff
@@ -355,8 +363,80 @@ private void RenderDataCell (BocColumnRenderingContext<TBocColumnDefinition> ren
 
       renderingContext.Writer.RenderBeginTag(HtmlTextWriterTag.Td);
 
+      var validationInfoText = renderingContext.Control.GetResourceManager().GetString(BocList.ResourceIdentifier.ValidationErrorInfoCell);
+      var validationFailures = renderingContext.Control.ValidationFailureRepository.GetUnhandledValidationFailuresForDataCell(
+          arguments.BusinessObject,
+          renderingContext.ColumnDefinition,
+          false);
+
+      if (validationFailures.Count > 0)
+      {
+        var tooltipStringBuilder = new StringBuilder();
+        tooltipStringBuilder.AppendLine(validationInfoText);
+        foreach (var validationFailure in validationFailures)
+        {
+          tooltipStringBuilder.Append(" • ");
+          tooltipStringBuilder.AppendLine(validationFailure.Failure.ErrorMessage);
+        }
+
+        renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Id, BocRowRenderer.GetCellIDForValidationMarker(renderingContext.Control, arguments.RowIndex, renderingContext.VisibleColumnIndex));
+        renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Title, tooltipStringBuilder.ToString());
+        renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Class, CssClasses.ValidationErrorMarker);
+        renderingContext.Writer.RenderBeginTag(HtmlTextWriterTag.Span);
```

Comment:
we need to annotate with aria-hidden (there's readymade const for this on HtmlAttriute2 and HtmlAriaHiddenAttributeValue